src/ui/hooks/workspaces_user.ts

```diff
@@ -23,50 +23,52 @@ import {
 } from "graphql/RejectPendingInvitation";
 import { WorkspaceUser, WorkspaceUserRole } from "ui/types";
 
-export function useGetWorkspaceMembers(workspaceId: string) {
-  const { data, loading, error } = useQuery<GetWorkspaceMembers, GetWorkspaceMembersVariables>(
-    gql`
-      query GetWorkspaceMembers($workspaceId: ID!) {
-        node(id: $workspaceId) {
-          ... on Workspace {
-            id
-            members {
-              edges {
-                node {
-                  ... on WorkspacePendingEmailMember {
-                    __typename
-                    id
-                    roles
-                    email
-                    createdAt
-                  }
-                  ... on WorkspacePendingUserMember {
-                    __typename
-                    id
-                    roles
-                    user {
-                      id
-                      name
-                      picture
-                    }
-                  }
-                  ... on WorkspaceUserMember {
-                    __typename
-                    id
-                    roles
-                    user {
-                      id
-                      name
-                      picture
-                    }
-                  }
+export const GET_WORKSPACE_MEMBERS = gql`
```

Comment:
Had to extract this GQL so I could use it with the imperative Apollo API too, because hooks can't be called conditionally and both `recording` and `recording.workspace` are nullable 😞

Reply:
wow - this gql looks super verbose. Also where are the types? I thought they were built?

Reply:
The types are here:
https://github.com/replayio/devtools/blob/main/src/graphql/GetWorkspaceMembers.ts

I found them really confusing and difficult to work with.

Reply:
oh yeah - these are auto generated

Reply:
Apparently we could improve the generated output by changing something on the backend/server but I'm not super familiar with GraphQL.